README.md

```diff
@@ -21,62 +21,22 @@ Amazon CloudWatch Agent uses the open-source project [telegraf](https://github.c
 ### Troubleshooting
 * [Troubleshooting CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/troubleshooting-CloudWatch-Agent.html)
 
-## Building and Running from source
-
-* Install go. For more information, see [Getting started](https://golang.org/doc/install)
-* The agent uses go modules for dependency management. For more information, see [Go Modules](https://github.com/golang/go/wiki/Modules)
-
-* Install rpm-build
-```
-sudo yum install -y rpmdevtools rpm-build
+## Building and running Amazon CloudWatch Agent 
+Use the following instructions to build and run Cloudwatch Agent:
+* To build Amazon CloudWatch Agent's binaries, you must [install go]((https://golang.org/doc/install)) and use `make build` to build the agent on Linux, Debian, Windows, Darwin. For more information on requirements,building and packaging binaries, please follow [this document](docs/build/build-run-package-cwagent-binaries.md)
```

Comment:
I still disagree with breaking out all of this info into another readme because I want specific steps in the main readme that tell me everything I need to get started, and this readme now lacks that info, but I will defer to others.

Reply:
I would say yes based on `my latest build`. Mostly I would like to introduce basic levels and delve dive into specific needs when developers need to (which would need a specific doc). However, in your opinion, what do you see lacking in the `README.doc`?